apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -225,3 +255,118 @@ function PopoverEndDate() {
     </p>
   `.toString();
 }
+
+function CreateCourseInstanceModal({
+  courseShortName,
+  csrfToken,
+  initialStartDateFormatted,
+  initialEndDateFormatted,
+  timezone,
+}: {
+  courseShortName: string;
+  csrfToken: string;
+  initialStartDateFormatted: string;
+  initialEndDateFormatted: string;
+  timezone: string;
+}) {
+  return Modal({
+    id: 'createCourseInstanceModal',
+    title: 'Create course instance',
+    formMethod: 'POST',
+    body: html`
+      <div class="form-group">
+        <label for="long_name">Long Name</label>
```

Comment:
Nit, we typically use sentence case for input labels. Can you change the others as well?

```suggestion
        <label for="long_name">Long name</label>
```